picamera/camera.py

```diff
@@ -2651,14 +2651,18 @@ def _set_video_frame_rate(self, value):
         assert type(value) in (int, float, fractions.Fraction)
         if type(value) != fractions.Fraction:
             value = fractions.Fraction(float(value))
+        # Need to limit the denominator, to something reasonable, I
+        # assume due to overflow.  Can't find documentation on this,
+        # but 512 seems to work OK.
+        value = value.limit_denominator(512)
```

Comment:
As per https://www.raspberrypi.org/forums/viewtopic.php?f=43&t=48238&start=75#p908303, it's in 1/256th steps at the lowest level. There's going to be no gain in working in smaller steps than that.